storage/connect/reldef.cpp

```diff
@@ -533,7 +575,9 @@ PTABDEF OEMDEF::GetXdef(PGLOBAL g)
 #endif // 0
 
   // Is the library already loaded?
+#if !defined(_AIX)
   if (!Hdll && !(Hdll = dlopen(soname, RTLD_NOLOAD)))
+#endif
```

Comment:
RTLD_NOLOAD is glibc specific and glibc dlopen() is fine about loading the same library multiple times. I'd suggest to remove this dlopen(RTLD_NOLOAD).